README.md

```
@@ -0,0 +1,439 @@
+
+pip install poetry
+poetry install
+
+poetry run uvicorn main:app --reload --port 8000
+```
+
+Now, follow the technical workflow to set up the **Airbyte** connection to the **FastAPI** source. Sync the data to **PostgreSQL** and set up the **MindsDB** connection to the **PostgreSQL** database. 
+
+Then proceed to train the model using the SQL commands provided in the **Prediction** section.
+
+**Airbyte Connection**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)